_site/README.md

```diff
@@ -0,0 +1,16 @@
+# bsadd-home
```

Comment:
This is more of a configuration question: Do we need to add _site folder to git ?

Reply:
Github builds jekyll itself ( tested that). Not sure whether we can get rid of _site folder altogether. Will let you know.